nexus/types/src/external_api/params.rs

```diff
@@ -2119,6 +2078,19 @@ pub struct SwitchPortApplySettings {
     pub port_settings: NameOrId,
 }
 
+/// Select a switch port by name
```

Comment:
Check if this doc string needs to be updated, as there are other fields that are not just name.